.babelrc

```diff
@@ -1,7 +1,8 @@
 {
-	"presets":
-		[["env", {
-			"targets": "es5"
-		}]],
-	"plugins": ["transform-object-rest-spread"]
+  "presets": [
+    ["env", {
+        "useBuiltIns": true
+    }]
+  ],
+  "plugins": [["transform-runtime", { "polyfill": false }], "transform-object-rest-spread"]
```

Comment:
this `{ "polyfill": false }` is a workaround to get it to not include `import` statements in the compiled code: https://github.com/babel/babel/issues/2877#issuecomment-270700000